test/components/auto_complete_test.rb

```diff
@@ -5,99 +5,120 @@
 class PrimerAutoCompleteTest < Minitest::Test
   include Primer::ComponentTestHelpers
 
-  def test_raises_if_no_input_is_defined
+  def test_raises_if_no_label_text_is_passed_in
     assert_raises ArgumentError do
       render_inline Primer::Beta::AutoComplete.new(src: "/url", input_id: "test-input", list_id: "my-list-id") do |component|
         component.icon(icon: :person)
       end
     end
   end
 
-  def test_renders_results_without_explicitly_calling_it
-    render_inline Primer::Beta::AutoComplete.new(src: "/url", input_id: "test-input", list_id: "my-list-id") do |component|
-      component.label(classes: "")
-      component.input(id: "test-input", name: "input")
+  def test_renders_label_without_explicitly_calling_it
+    render_inline Primer::Beta::AutoComplete.new(label_text: "Cool Fruits", src: "/url", input_id: "test-input", list_id: "my-list-id")
+
+    assert_selector("auto-complete[for=\"my-list-id\"][src=\"/url\"]") do
```

Comment:
I think it's permissible to wrap these strings in single quotes because otherwise you have to escape all the double quotes.

```suggestion
    assert_selector('auto-complete[for="my-list-id"][src="/url"]') do
```